hal/include/hal/PinNameAliases.h

```diff
@@ -0,0 +1,108 @@
+
+/** \addtogroup hal */
+/** @{*/
+/* mbed Microcontroller Library
+ * Copyright (c) 2020 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef MBED_PIN_NAME_ALIASES_H
+#define MBED_PIN_NAME_ALIASES_H
+
+/* Aliases for legacy reasons. To be removed in the next Mbed OS version */
+#if defined (CONSOLE_TX) && (CONSOLE_RX)
```

Comment:
CONSOLE_TX and CONSOLE_RX are defined as enum,
So #if defined will be always "false"